client/src/components/Results/Cards.js

```diff
@@ -241,35 +241,21 @@ class Card extends React.Component {
                     : null}
                 </div>
               </div>
-
-            
-                 <div className="card-header">
-            <div className="card-category">{textMap.parsedCategory}</div>
-            {textMap.parsedCOVID === "CLOSED DUE TO COVID" ? (
-              <div className="covid-item">{textMap.parsedCOVID}</div>
-            ) : null}
-          </div>
+              <div className="card-content">
+                {textMap.parsedCOVID === "CLOSED DUE TO COVID" ? (
+                  <div className="covid-item">CLOSED</div>
+                ) : (
+                  textMap.parsedHours
+                )}
+              </div>
+            </div>
+          ) : null}
+             {!(textMap.parsedEmergency ==="Lorem ipsum dolor sit amet, consectetur...") ? (
```

Comment:
This conditional statement should be something like `!(textMap.parsedEmergency === "")` or it will always evaluate to true.